test.html.haml

```diff
@@ -0,0 +1,2 @@
+= some_long_line  |
```

Comment:
Don't use the `|` character to split up lines. Wrap on commas or extract code into helper.